test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -217,61 +250,7 @@ def forward(self, x):
                 return x
 
         x = torch.randn(2, 1)
-        self.run_model_test_with_external_data(torch.jit.script(LargeModel()), x)
```

Comment:
A number of these model tsts are not present in the test_models file. 
Why are we removing all of these?